File: app/app/Model/Member/Response/MemberGetMyResponse.swift
struct MemberGetMyResponse: Codable, Identifiable {

    let memberId: Int64
    var publicImages: [MemberGetImageResponse]
    var privateImages: [MemberGetImageResponse]
    var nickname: String
    var birthYear: Int
    var bio: String?

    var id: Int64 { memberId }
}

struct MemberGetImageResponse: Codable, Identifiable {

    let imageId: Int64
    let url: String
    let key: String
    let type: String
    let sortOrder: Int

    var id: Int64 { imageId }
}

File: server/src/main/kotlin/com/openmpy/server/member/dto/response/MemberGetMyResponse.kt
package com.openmpy.server.member.dto.response

import com.openmpy.server.image.domain.type.MemberImageType

data class MemberGetMyResponse(

    val memberId: Long,
    val publicImages: List<MemberGetImageResponse>,
    val privateImages: List<MemberGetImageResponse>,
    val nickname: String,
    val birthYear: Int,
    val bio: String?,
)

data class MemberGetImageResponse(

    val imageId: Long,
    val url: String,
    val key: String,
    val type: MemberImageType,
    val sortOrder: Int
)